packages/app/src/components/contract/ContractInfoTab.vue

```diff
@@ -270,23 +278,5 @@ const tabs = computed(() => {
 <style lang="scss" scoped>
 .functions-contract-container {
   @apply mt-4;
```

Comment:
Why have the styles below been moved? Existing tabs are broken (most probably because of this change):
[Current version on prod](https://sepolia.explorer.zksync.io/address/0xCcaD58036a321C0Bcce62514269c9DD5D5135a8b#contract#read) 
<img width="1274" alt="image" src="https://github.com/user-attachments/assets/a38b35a1-a063-4f4a-aae1-08be1186d6e0" />

[This version locally](http://localhost:3010/address/0xCcaD58036a321C0Bcce62514269c9DD5D5135a8b#contract#read)
<img width="1272" alt="image" src="https://github.com/user-attachments/assets/b04ec058-d41c-42b0-89ec-7aa7d9f457b8" />